ssl/d1_clnt.c

```diff
@@ -128,38 +128,25 @@
 # include <openssl/dh.h>
 #endif
 
-static const SSL_METHOD *dtls1_get_client_method(int ver);
 static int dtls1_get_hello_verify(SSL *s);
 
-static const SSL_METHOD *dtls1_get_client_method(int ver)
-{
-    if (ver == DTLS_ANY_VERSION)
-        return DTLS_client_method();
-    else if (ver == DTLS1_VERSION || ver == DTLS1_BAD_VER)
-        return DTLSv1_client_method();
-    else if (ver == DTLS1_2_VERSION)
-        return DTLSv1_2_client_method();
-    else
-        return NULL;
-}
```

Comment:
So it looks like those function also become unused in 1.0.2, but maybe we should keep them around and try to minimize the changes.